### PR TITLE
Migrated MSSQLConfigStore to @configu/integrations #520

### LINKED ISSUE #520: Migrate `MSSQLConfigStore` to `@configu/integrations`
### Suggestion

Migrate the `MSSQLConfigStore` from its current setup to the new `@configu/integrations` module as part of the major restructuring. The goal is to move all third-party integrations, including stores, expressions, and libraries, to `@configu/integrations`.

### Motivation

This migration is part of a larger effort to better organize and modularize the product. Moving the `MSSQLConfigStore` will align with the new structure, making the system cleaner and more maintainable.

### Context

You can refer to the already migrated stores for guidance:  
[Migrated stores in integrations](https://github.com/configu/configu/tree/main/packages/integrations/stores).

Base stores implementations from `@configu/ts` can be found here:  
[Base stores in TS](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/ts/src/stores).

The old node-based stores were previously in `@configu/node`:  
[Node stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/node/src/stores).

The old browser-based stores were previously in `@configu/browser`:  
[Browser stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/browser/src/stores).

### PR DESCRIPTION
**Pull request by Mila2999**
Closes #520

### CODE REVIEW

**Comment by Mila2999** [comment]
Hi @rannn505, i see that healthcare ci failed. what does it mean?

File: packages/integrations/stores/ms-sql/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@configu-integrations/json-file",
```
Comment (davidmeirlevy) [inline packages/integrations/stores/ms-sql/package.json:2]: please fix the package name according the store name.

File: packages/integrations/stores/ms-sql/src/ms-sql.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { type SqlServerConnectionOptions } from 'typeorm/driver/sqlserver/SqlServerConnectionOptions.js';
```
Comment (davidmeirlevy) [inline packages/integrations/stores/ms-sql/src/ms-sql.ts:1]: you should probably add "typeorm" as a dependency.

**Review by davidmeirlevy — CHANGES_REQUESTED** [review]
@Mila2999 thanks for your PR!
just a few fixes and we can merge it. 👍

**Comment by Mila2999** [comment]
Hi @davidmeirlevy, I made all the changes you requested. I see ci still failing, any idea why?:)

**Review by davidmeirlevy — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
@Mila2999 don't worry, that's a lockfile issue.
we are fixing this file at the end of the day.

merged 🚀

**Pull request merged**